js/src/controls/SearchControl.js

```diff
@@ -50,8 +51,13 @@ export class LeafletSearchControlView extends control.LeafletControlView {
 
   leaflet_events() {
     if (this.model.get('layer') !== null) {
+        var found_style = this.model.get('found_style');
         this.obj.on('search:locationfound', function(e) {
-            e.layer.setStyle({fillColor: '#3f0', color: '#0f0'});
+            if (Object.keys(found_style).length != 0) {
+                e.layer.setStyle(found_style);
+            } else {
+               e.layer.setStyle({fillColor: '#3f0', color: '#0f0'});
```

Comment:
It might be nicer to have this as the default value for `found_style` Python side. So that if people wants the found feature to not be styled (and only use the marker) then they can set `found_style` to empty dict and there is no extra styling. What do you think?

Reply:
yes, this makes sense.  I will do the required change.